test/index.test.js

```diff
@@ -19,7 +20,11 @@ const sampleOnlyDeletions = require('./fixtures/pull_request.deletions.json')
 const sampleDelAddModif = require('./fixtures/pull_request.deletions.modif.add.json')
 
 function mockPRContents (github, PR) {
-  github.pullRequests.listFiles = jest.fn().mockResolvedValue(PR)
+  github.pullRequests.listFiles = jest.fn(() => {
+    --github.numPagesOfContent
+    return PR
```

Comment:
We shouldn't use variables in all caps.  It effectively looks like a const

Reply:
It used to be with big letters before but you are right.